### PR TITLE
CBP3461 Immediate Delivery

### PR DESCRIPTION
**Pull request by nissimsan**

### CODE REVIEW

File: docs/openapi/components/schemas/common/ImmediateDelivery.yml [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+    enum:
+      - IRS
+      - SSN
+      - CBP
+    $linkedData:
+      term: assignedIdentifierType
+      '@id': https://w3id.org/traceability#assignedIdentifierType
+  entryNumber:
+    title: Entry Number
+    description: The 11 digit alphanumeric code. The entry numberis comprised of the three-digit filer code, followed by the seven-digit entry number, and completed with the one-digit check digit. The Entry Filer Code represents the three-character alphanumeric filer code assigned to the filer or importer by CBP. The Entry Number represents the seven-digit number assigned by the filer. The number may be assigned in any manner convenient, provided that the same number is not assigned to more than one CBP Form 3461. Leading zeros must be shown. The check digit is computedon the previous 10 characters. The formula for calculating the check digit can be found in Appendix 1.
```
Comment (TallTed) [inline docs/openapi/components/schemas/common/ImmediateDelivery.yml:53]:
```suggestion
    description: The 11 digit alphanumeric code. The entry number is comprised of the three-digit filer code, followed by the seven-digit entry number, and completed with the one-digit check digit. The Entry Filer Code represents the three-character alphanumeric filer code assigned to the filer or importer by CBP. The Entry Number represents the seven-digit number assigned by the filer. The number may be assigned in any manner convenient, provided that the same number is not assigned to more than one CBP Form 3461. Leading zeros must be shown. The check digit is computed on the previous 10 characters. The formula for calculating the check digit can be found in Appendix 1.
```

File: docs/openapi/components/schemas/common/ImmediateDelivery.yml [hunk 2]
```diff
@@ -0,0 +1,340 @@
...
+  portOfUnlading:
+    title: Port Of Unlading
+    description: Record the U.S. port code where the merchandise was unladefrom the importing vessel, aircraft or train. Do not providethe name of the port instead of the numeric code. For merchandise arriving in the U.S. by means of transportation other than vessel, rail or air, leave blank. For merchandise arriving in the customs territory from a U.S. FTZ, leave blank.
+    $ref: ./Place.yml
+    $linkedData:
+      term: portOfUnlading
+      '@id': https://schema.org/Place
+  transportMode:
+    title: Mode Of Transportation
+    description: The Mode of Transportationfrom the available options Air, Ocean, Rail, Truck, Hand Carry, Pipeline, or Other.
```
Comment (TallTed) [inline docs/openapi/components/schemas/common/ImmediateDelivery.yml:131]:
```suggestion
    description: The Mode of Transportation from the available options Air, Ocean, Rail, Truck, Hand Carry, Pipeline, or Other.
```

File: docs/openapi/components/schemas/common/ImmediateDelivery.yml [hunk 3]
```diff
@@ -0,0 +1,340 @@
...
+    type: string
+    $linkedData:
+      term: bolType
+      '@id': https://w3id.org/traceability#bolType
+  scac:
+    title: Standard Carrier Alpha Code
+    description: For merchandise arriving in the U.S. by air, record the two digit IATA alpha code corresponding to the name of the airline which transported the merchandise from the last airport of foreign lading to the first U.S. airport of unlading.  If the carrier filedoes not contain a specific air carrier’s code, write the designation “*C” for Canadian airlines, “*F” for other foreign airlines, and “*U” for U.S. airlines.  These designations should be used only for unknown charter and private aircraft.  When a private aircraft is being entered under its own power (ferried), the designation “**” will be used. For merchandise arriving in the U.S. by means of transportation other than vessel or air, leave blank.  Do not record the name of a domestic carrier transporting merchandise after initial unlading in the U.S. For merchandise arriving in the customs territory from a Foreign Trade Zone (FTZ), insert “FTZ” followed by the FTZ number.  Use the following format FTZ NNNN.
```
Comment (TallTed) [inline docs/openapi/components/schemas/common/ImmediateDelivery.yml:211]:
```suggestion
    description: For merchandise arriving in the U.S. by air, record the two digit IATA alpha code corresponding to the name of the airline which transported the merchandise from the last airport of foreign lading to the first U.S. airport of unlading.  If the carrier file does not contain a specific air carrier’s code, write the designation “*C” for Canadian airlines, “*F” for other foreign airlines, and “*U” for U.S. airlines.  These designations should be used only for unknown charter and private aircraft.  When a private aircraft is being entered under its own power (ferried), the designation “**” will be used. For merchandise arriving in the U.S. by means of transportation other than vessel or air, leave blank.  Do not record the name of a domestic carrier transporting merchandise after initial unlading in the U.S. For merchandise arriving in the customs territory from a Foreign Trade Zone (FTZ), insert “FTZ” followed by the FTZ number.  Use the following format FTZ NNNN.
```

**Review by OR13 — APPROVED** [review]

**Comment by OR13** [comment]
please address merge conf.

**Review by BenjaminMoe — APPROVED** [review]

**Review by mkhraisha — APPROVED** [review]

**Review by mprorock — APPROVED** [review]

**Pull request merged**